apps/coordinator/src/components/Wallet/TransactionAnalysis/SWASlider.tsx

```diff
@@ -0,0 +1,227 @@
+      });
+
+      // Prepare parameters for waste calculation
+      const coinAmounts = inputs.map((input: any) => {
+        const amountSats = input.amountSats
+          ? new BigNumber(input.amountSats)
+          : new BigNumber(bitcoinsToSatoshis(input.amount));
+        return amountSats.toNumber();
+      });
+
+      const hasChange = outputs.some(
+        (output: any) => output.address === changeAddress,
+      );
+
+      // Calculate waste using the health package function
+      const newWasteAmount = calculateWasteMetric({
+        coinAmounts,
+        config: {
+          scriptType: walletConfig.addressType,
+          requiredSigners: walletConfig.quorum.requiredSigners,
+          totalSigners: walletConfig.quorum.totalSigners,
+        },
+        effectiveFeeRate: feeRateSatPerVb.toNumber(),
+        estimatedLongTermFeeRate: longTermFeeEstimate,
+        hasChange,
+        spendAmount: nonChangeOutputsTotalSats.toNumber(),
+      });
+
+      setWasteAmount(new BigNumber(newWasteAmount));
+    } catch (err) {
+      console.error("Error calculating waste metrics:", err);
+      setWasteAmount(new BigNumber(0));
+    }
+  }, [
+    fee,
+    feeRate,
+    outputs,
+    inputs,
+    changeAddress,
+    walletConfig,
+    longTermFeeEstimate,
+  ]);
+
+  const feeLevelInfo = useMemo(
+    () => getFeeLevelInfo(longTermFeeEstimate),
+    [longTermFeeEstimate],
+  );
+
+  return (
+    <Box className="swa-slider-container">
+      {/* Header */}
+      <Typography variant="h6" className="waste-analysis-title">
+        Waste Analysis
+      </Typography>
+      <Typography variant="body2" className="waste-analysis-description">
+        Analyzes if spending Bitcoin now is efficient or if waiting could be
+        cheaper
+      </Typography>
+
+      {/* Fee Waste Box */}
+      <Box className="potential-fee-waste-container">
+        <Typography variant="subtitle1" className="potential-fee-waste-title">
+          Potential Fee Waste
+        </Typography>
+
+        {/* Slider */}
+        <Box className="slider-container">
+          <Box className="gradient-track" />
+          <Slider
+            value={longTermFeeEstimate}
+            min={1}
+            max={500}
+            step={1}
+            onChange={handleSliderChange}
+            aria-labelledby="long-term-fee-estimate-slider"
+            className="custom-slider"
+            valueLabelDisplay="auto"
+            getAriaValueText={valuetext}
+            valueLabelFormat={valuetext}
+          />
+        </Box>
+
+        {/* Waste Amount */}
+        <Box className="waste-amount-container">
+          <Typography variant="h4" className="waste-amount-text">
+            {formatNumber(wasteAmount)}
+          </Typography>
+          <Typography variant="body2" className="waste-amount-units">
+            sats
+          </Typography>
+          <Tooltip title="Shows how much you might waste in fees by spending now instead of waiting for lower fees">
```

Comment:
We're not creating waste in fees. We're just generating (or destroying) waste for our wallet over time.
 
```suggestion
          <Tooltip title="Shows how much waste this transaction will create or destroy (if negative) for your wallet if spent now depending on your long term fee rate expectations">
```